src/main/java/ch/njol/skript/lang/Variable.java

```diff
@@ -119,15 +119,10 @@ public static boolean isValidVariableName(String name, boolean allowListVariable
 				/*
 				A lot of people already use '-' so we want to skip this warning iff they're using it here
 				*/
-				if (first == '-') {
-					for (VariablesStorage store : Variables.getStores()) {
-						@Nullable Pattern pattern = store.getNamePattern();
-						if (pattern != null && pattern.pattern().equals("(?!-).*"))
-							continue check_reserved_tokens;
-					}
-				}
-				Skript.warning("The character '" + token + "' is reserved at the start of variable names, " +
-					"and may be restricted in future versions");
+				if (first == '-')
```

Comment:
is the label for this loop still needed?

Reply:
Probably not.